src/exercises/01-Query-Renderer/README.md

```diff
@@ -245,53 +254,56 @@ Arguments can be passed into the `query` of a `QueryRenderer` through the `varia
 💻 _Add a variable to the `query` prop named `artistID`:_
 
 ```typescript
-    <QueryRenderer<Artist1QueryRendererQuery>
-      // ...
-      query={graphql`
+<QueryRenderer<Artist1QueryRendererQuery>
+  // ...
+  query={graphql`
         query Artist1QueryRendererQuery($artistID: ID!) {
           artist(id: $artistID) {
             // ...
           }
         }
       `}
-      // ...
-    />
+  // ...
+/>
 ```
+
 _./Artist1QueryRenderer.tsx_
 
-Notice that we add the variable to the top-level `query` as well as the `artist` type we're querying. 
+Notice that we add the variable to the top-level `query` as well as the `artist` type we're querying.
 
 At this point you'll get a type error on the `variables` prop. This is because the query is expecting a property named `artistID` on the `variables` argument.
 
 💻 _Add a hardcoded `artistID` to the `variables` prop:_
 
 ```typescript
-    <QueryRenderer<Artist1QueryRendererQuery>
-      // ...
-      variables={{artistID: '1'}}
-      // ...
-    />
+<QueryRenderer<Artist1QueryRendererQuery>
+  // ...
+  variables={{ artistID: "1" }}
```

Comment:
This is a nice fix to a code-block inside a .md, but I plan to take an extra scrutinous eye to the rest of the changes to make sure it doesn't do anything unexpected in the existing exercise instructions.